### ISSUE TITLE
Crash after a few minutes of playing

### ISSUE THREAD

**Issue by ML98** [Cazadorsniper/PO3#1280]
# Use the search function before reporting to see if the issue has been reported before.
**Pack Version:3.3.59**
**Pack Mode (Normal, Titan, Kappa):Normal**
**Singleplayer, Multiplayer, or LAN?:Singleplayer (via Twitch launcher)**

**Description:On multiple occasions I experienced periods of complete game freeze while interacting with inventories or when the world was doing the usual backup. It didn't always happen, but recently I cannot play for more than 20 minutes or so that I go to close my inventory with E and then freeze. Sometimes I wait for it to unfreeze and I go on playing, until it freezes again and completely shuts down.

At first I thought it was my pc, because I am on a laptop, but I have 12 Gb of RAM and I allocated 7516 for PO3 with the launcher. When I play I have all the other programs shut down, and I'm running only PO3.
I installed antiviruses, updated all the drivers, everything with my pc is great, infact if I play normal Minecraft I don't experience those issues, not even lag (granted this is a huge modpack and it is different, I get it, but at least if it was a machine problem, I should feel it elsewhere and I don't).

I haven't even progressed a lot into the pack, in fact I was just building my new ME System after I finally got plastic from Magneticraft, super early, and while I was doing that I even destroyed all my previous power generation because I wanted to move it in the new room, but it still crashed.
My mob system is shut off, my mystical agricolture farm is shut off, everything was shut off. No lag at all. It just went and boom. It crashed.

I don't know if this has already been posted, if so I apologize, but I couldn't find the exact same issue.

I will leave here the latest log using pastebin, but I have other logs which I will add with no problem if you need them, because this one is pretty short and shows only that I tried to sleep in the air with a sleeping bag and the whilst doing the second backup it died (it froze after I closed my inventory with E).

Thank you in advance**

**Steps to reproduce: I don't really know how you would reproduce it... Maybe if you try a couple of times with opening and closing inventories, but I don't know. Question: could it be an interaction between Magneticraft shelving units and chests or my dank null or quantum bags? Those are all the inventories I have access to.**

**Crash log (use pastebin or issue will be closed):https://pastebin.com/j3etX2Sv**

**Comment by Cazadorsniper** [comment]
I can't tell what the issue is with that, I need the entire latest.log from the logs folder

**Comment by ML98** [comment]
I can assure you I gave you the latest log. If you want the past logs I can post them, just tell me how (do you want them rar or all pastebin) and I'll post them. Thank you for answering

**Comment by Cazadorsniper** [comment]
Latest.log usually has more information that.

**Comment by ML98** [comment]
I don't know what to say, really. That is all I have in the latest log, but I will paste a gist of the previous log, which was the freeze and crash I had right before the one which made me write this issue.

https://gist.githubusercontent.com/ML98/13d34d4f2bacf88ee65fb16f6b88c954/raw/0f755f7b2334ce3304790209feec592181d70d35/2020-06-23-3

Sorry if I use a gist, but I tried with pastebin, but it says it's too big. The only things I modified from it are my PC Username, and my MC username.

**Comment by Cazadorsniper** [comment]
Does your laptop have a video card?

**Comment by Cazadorsniper** [comment]
Is your minecraft name really playername? lol

**Comment by Cazadorsniper** [comment]
Also it seems that some conduits couldn't be harvested correctly: 
Block harvest failed unexpectedly.
Block: enderio:block_conduit_bundle[opaque=false]
Tool: tconstruct.pickaxe@0
Pos: BlockPos{x=467, y=82, z=-399}

Block harvest failed unexpectedly.
Block: enderio:block_conduit_bundle[opaque=false]
Tool:.tconstruct.pickaxe@0
Pos: BlockPos{x=467, y=83, z=-399}

Block harvest failed unexpectedly.
Block: enderio:block_conduit_bundle[opaque=false]
Tool: tconstruct.pickaxe@0
Pos: BlockPos{x=469, y=83, z=-400}

**Comment by Cazadorsniper** [comment]
You can try using the forge.cfg configs to remove erroring entities/tileentities.

**Comment by ML98** [comment]
Lmao, I changed my MC name to Player Name from the log, I guess useless privacy.
Those conduits feel weird, it looks like I was harvesting a bunch of conduits from a sort of grid and, one in a position, one above it and the other one in an L shaped position away. There must have been other conduits in between, because I remember that I didn't misplace any conduit. So some have registered and some didn't.

I think I have a video card, if I interpreted the term correctly I have a 2047Mb Nvidia GeForce 940MX and an Intel HD Graphics 620. I don't know if it is what you asked tho lol.

The last step, with the forge.cfg I changed it, but I will test it tonight because right now I can't. Could you explain a bit more what would happen now that I changed that to true if an entity gives me an error? How could it "screw up everything"? Thank you for answering

**Comment by Cazadorsniper** [comment]
The forge config just means anything that errors will simply disappear, so if an me controller broke then it will simply disappear causing the rest of the me system to not work because of lack of the controller.

Try getting minecraft to use the Nvidia GeForce 940MX because its using the Intel HD Graphics 620.

**Comment by ML98** [comment]
So, I tried everything you suggested, I got Minecraft to only use my 940MX, and I even got an increase in FPS as expected, but then when it started the first world backup it completely stopped.
I was looking in the TIC book for the chisel, and then the backup started. It froze for anout 5 minutes. In the latest log which I'll post as a pastebin it says backup done in 00:01 seconds, that is not true at all, my PC froze and then, when it seemed it had completed it, I closed the book with ESC and I wanted to go around and check if some blocks broke as you said it could happen around the world, due to that change in the config files, but as soon as I started moving, I got a Windows error and the game shut down.

It seems like a never ending issue, but I don't believe it is my PC, because for those minutes I could play I had little to no lag, it felt like butter and then boom, it just kinda imploded with no fricking reason.

I will get this opportunity to say how much I love the PO3 pack, even if I'm playing it like stroke after stroke lol. It is really big and really really good. I understand there might be some nasty issues here and there, but even if it's a bummer that I can't play it properly, every time I can, I always have a great time experiencing with new mods and having thousands of resources from my wonderful seeds.

Here is the pastebin from my latest log, thanks again

https://pastebin.com/ExZHwU8K

**Comment by Cazadorsniper** [comment]
Maybe your storage is full? Try turning off the backups in the ftbbackups folder.

**Comment by ML98** [comment]
Nope, more than 450  Gb free. I'll try turning them off and I'll let you know.

**Comment by ML98** [comment]
So, I tried once again, and this time it was so much worse. After disabling the server backups, it ran smoothly for the usual 10 - 15 minutes and then when I decided to go grab basalz from the deep dark, I was flying with my angel ring and total freeze. Not only MC, the whole PC is frozen.

It has been frozen for 10 minutes then it went black screen and then after other 5 minutes it went to the desktop.
The desktop was frozen, so I tried to close it manually with task manager. It failed and now it is still frozen.
Even if we conclude that the last (just the last task manager stuff is my fault because I did it), the rest is a serious game issue, which doesn't allow me to play at all.

After all the analysis and corrections I applied after your advices, and looking at the logs, the only reasonable conclusion is that the game has a sort of memory issue which if failed to store, it will create a chain of conflicts with other internal stuff that totally annhilites Minecraft.

Update: I was writing from my smartphone because PC was frozen, but now it got back up. It took 15 minutes. 

I reopened twitch and just the launcher, but I cannot find the latest log, it just didn't create it in the folder.

In conclusion, I checked every possible aspect of the PC and it is working fine, there is no machine problem, it is software. 

Since I doubt I will get a different result without an update, I will have to stop playing the pack. I'm really sorry, but this is right now the only logical solution, but that doesn't mean I didn't enjoy it and I will come back as soon as the issue is fixed.

**Comment by Cazadorsniper** [comment]
Someone in the discord would like a copy of your world.
Another person said they had similar issues but it was because there Hard Drive or Motherboard was failing.

**Comment by Cazadorsniper** [comment]
Do you have any extra mods loaded?

**Comment by ML98** [comment]
Excuse me if I didn't comment;
1) I will post immediately the world in a .rar format on the discord, thank you.

2) nope, I'm using the twitch launcher, so if that isn't messed up I am playing the pack as intended.

**Comment by Cazadorsniper** [comment]
It is possible that twitchapp messed up.
